kodilla-stream/src/main/java/com/kodilla/stream/forumuser/ForumUser.java

```diff
@@ -0,0 +1,48 @@
+package com.kodilla.stream.forumuser;
+
+import java.time.LocalDate;
+
+public final class ForumUser {
+
+    private final int userId;
+    private final String userName;
+    private final char sex;
+    private final LocalDate birthDate;
+    private final int postsNumber;
+
+    public ForumUser(final int userId, final String userName, final char sex, final int yearOfBirth,
+                     final int monthOfBirth, final int dayOfBirth, final int postsNumber) {
+        this.userId = userId;
+        this.userName = userName;
+        this.sex = sex;
+        this.birthDate = LocalDate.of(yearOfBirth, monthOfBirth, dayOfBirth);
+        this.postsNumber = postsNumber;
+    }
+
+    public int getUserId() {
+        return userId;
+    }
+
+    public char getSex() {
+        return sex;
+    }
+
+    public LocalDate getBirthDate() {
+        return birthDate;
+    }
+
+    public int getPostsNumber() {
+        return postsNumber;
+    }
+
+    @Override
+    public String toString() {
+        return "ForumUser{" +
+                "userId=" + userId +
+                ", userName='" + userName + '\'' +
+                ", sex=" + sex +
+                ", birthDate=" + birthDate +
+                ", postsNumber=" + postsNumber +
+                '}';
+    }
```

Comment:
Ponieważ przechowywać będziemy obiekty w liście, to dobrą praktyką jest nadpisanie mtod equals i hashCode (tym bardziej, że będziesz obiekty potem wrzucał do HashMapy)